src/java.base/share/classes/java/io/ObjectStreamConstants.java

```diff
@@ -35,118 +35,118 @@ public interface ObjectStreamConstants {
 
     /**
      * Version number that is written to the stream header.
      */
-    static final short STREAM_VERSION = 5;
+    static short STREAM_VERSION = 5;
 
     /* Each item in the stream is preceded by a tag
      */
 
     /**
      * First tag value.
      */
-    static final byte TC_BASE = 0x70;
+    static byte TC_BASE = 0x70;
 
     /**
      * Null object reference.
      */
-    static final byte TC_NULL =         (byte)0x70;
+    static byte TC_NULL = 0x70;
 
     /**
      * Reference to an object already written into the stream.
      */
-    static final byte TC_REFERENCE =    (byte)0x71;
+    static byte TC_REFERENCE = 0x71;
 
     /**
      * new Class Descriptor.
      */
-    static final byte TC_CLASSDESC =    (byte)0x72;
+    static byte TC_CLASSDESC = 0x72;
 
     /**
      * new Object.
      */
-    static final byte TC_OBJECT =       (byte)0x73;
+    static byte TC_OBJECT = 0x73;
 
     /**
      * new String.
      */
-    static final byte TC_STRING =       (byte)0x74;
+    static byte TC_STRING = 0x74;
 
     /**
      * new Array.
      */
-    static final byte TC_ARRAY =        (byte)0x75;
+    static byte TC_ARRAY = 0x75;
 
     /**
      * Reference to Class.
      */
-    static final byte TC_CLASS =        (byte)0x76;
+    static byte TC_CLASS = 0x76;
 
     /**
      * Block of optional data. Byte following tag indicates number
      * of bytes in this block data.
      */
-    static final byte TC_BLOCKDATA =    (byte)0x77;
+    static byte TC_BLOCKDATA = 0x77;
 
     /**
      * End of optional block data blocks for an object.
      */
-    static final byte TC_ENDBLOCKDATA = (byte)0x78;
+    static byte TC_ENDBLOCKDATA = 0x78;
 
     /**
      * Reset stream context. All handles written into stream are reset.
      */
-    static final byte TC_RESET =        (byte)0x79;
+    static byte TC_RESET = 0x79;
 
     /**
      * long Block data. The long following the tag indicates the
      * number of bytes in this block data.
      */
-    static final byte TC_BLOCKDATALONG= (byte)0x7A;
+    static byte TC_BLOCKDATALONG = 0x7A;
 
     /**
      * Exception during write.
      */
-    static final byte TC_EXCEPTION =    (byte)0x7B;
+    static byte TC_EXCEPTION = 0x7B;
 
     /**
      * Long string.
      */
-    static final byte TC_LONGSTRING =   (byte)0x7C;
+    static byte TC_LONGSTRING = 0x7C;
 
     /**
      * new Proxy Class Descriptor.
      */
-    static final byte TC_PROXYCLASSDESC =       (byte)0x7D;
+    static byte TC_PROXYCLASSDESC = 0x7D;
 
     /**
      * new Enum constant.
      * @since 1.5
      */
-    static final byte TC_ENUM =         (byte)0x7E;
+    static byte TC_ENUM = 0x7E;
 
     /**
      * Last tag value.
      */
-    static final byte TC_MAX =          (byte)0x7E;
+    static byte TC_MAX = 0x7E;
 
     /**
      * First wire handle to be assigned.
      */
-    static final int baseWireHandle = 0x7e0000;
+    static int BASE_WIRE_HANDLE = 0x7e0000;
```

Comment:
Isn't it a part of public API? You shouldn't rename such things in cleanup PRs.

https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/io/ObjectStreamConstants.html#baseWireHandle

Reply:
Oops. Thanks for spotting this. Fixed.

Reply:
The other thing here is that dropping the final modifier is technically a signature change so if that is included then it will need to be tracked via a CSR.